AsyncDisplayKit/ASImageNode.mm

```diff
@@ -410,6 +415,174 @@ - (void)setImageModificationBlock:(asimagenode_modification_block_t)imageModific
   _imageModificationBlock = imageModificationBlock;
 }
 
+
+#if TARGET_OS_TV
+- (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
+{
+  [super touchesBegan:touches withEvent:event];
+  isDefaultState = NO;
+  UIView *view = [self getView];
+  CALayer *layer = view.layer;
+
+  CGSize targetShadowOffset = CGSizeMake(0.0, self.bounds.size.height/8);
+  [layer removeAllAnimations];
+  [CATransaction begin];
+  [CATransaction setCompletionBlock:^{
+    layer.shadowOffset = targetShadowOffset;
+  }];
+  
+  CABasicAnimation *shadowOffsetAnimation = [CABasicAnimation animationWithKeyPath:@"shadowOffset"];
+  shadowOffsetAnimation.toValue = [NSValue valueWithCGSize:targetShadowOffset];
```

Comment:
Ideally abstract this to something like AsyncDisplayKit+tvOS or something.  Not sure what else we could move there but there's quite a bit of animation code that hopefully would be movable.